src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_SpargeTower.java

```diff
@@ -224,6 +224,11 @@ public boolean isCorrectMachinePart(ItemStack aStack) {
                             tRecipe.mFluidInputs[1]);
                     this.mOutputFluids = aFluidOutputs.toArray(new FluidStack[0]);
                     updateSlots();
+
+                    if (lEUt > 0) {
```

Comment:
Since this value is stored in existing machines, modify `loadNBTData` to also make it negative with comment